jumpscale/packages/admin/actors/admin.py

```diff
@@ -140,5 +149,19 @@ def clear_blocked_nodes(self) -> str:
         j.sals.reservation_chatflow.reservation_chatflow.clear_blocked_nodes()
         return j.data.serializers.json.dumps({"data": "blocked nodes got cleared successfully."})
 
+    @actor_method
+    def get_notifications(self) -> str:
+        notifications = []
+        if j.tools.notificationsqueue.count() >= 10:
```

Comment:
I think this must be less than or equal to ten

Reply:
No we need to fetch all new notifications if they are > 10, else we would fetch 10 notifications (new + old)